.github/workflows/build.yml

```diff
@@ -73,3 +73,9 @@ jobs:
         env:
           QUAY_USERNAME: ${{ secrets.QUAY_USERNAME }}
           QUAY_PASSWORD: ${{ secrets.QUAY_PASSWORD }}
+
+  all_passed:
+    needs: [build_manylinux]
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "All jobs passed"
```

Comment:
Hey @mayeut, I just saw this and realized that it's an incorrect and **dangerous** implementation which allows merging PRs on failures. Please, see the full explanation here: https://github.com/marketplace/actions/alls-green#why.